app/src/main/java/com/parishod/watomatic/NotificationService.java

```diff
@@ -26,8 +26,8 @@
     private final String TAG = NotificationService.class.getSimpleName();
     CustomRepliesData customRepliesData;
     private WhatsappAutoReplyLogsDB whatsappAutoReplyLogsDB;
-    private final int DELAY_BETWEEN_REPLY_IN_MILLISEC = 20 * 1000;
-    private final int DELAY_BETWEEN_NOTIFICATION_RECEIVED_IN_MILLISEC = 60 * 1000;
+    private final int DELAY_BETWEEN_REPLY_IN_MILLISEC = 10 * 1000;
+    private final int DELAY_BETWEEN_NOTIFICATION_RECEIVED_IN_MILLISEC = 50 * 1000;
```

Comment:
Aweseome @pixincreate ! 😃

`private final int DELAY_BETWEEN_NOTIFICATION_RECEIVED_IN_MILLISEC = 50 * 1000;`

This variable/member is not named well which might have confused you. What this actually does is to check time between the notifications sent time and actual received time. And if the notification is older than 60 seconds, it will not reply. This prevents replying to older notifications.

This should actually be 60 or more otherwise there is a possibility that it will miss new notifications. If you can, may be change it to 120 as shown below or leave it as is.

`private final int DELAY_BETWEEN_NOTIFICATION_RECEIVED_IN_MILLISEC = 120 * 1000;`

Otherwise its okay, I will change it and accept the PR.

Thank you for your first contribution 🎉
I will make necessary changes (or you can too) and will test it and merge in a few hours.